marketdepthviewer/marketdepthviewer.go

```diff
@@ -19,18 +20,37 @@ import (
 	"google.golang.org/grpc"
 )
 
+// Opts command line options
+type Opts struct {
+	Market     string
+	ServerAddr string
+	UseDeltas  bool
+}
+
+// MarketDepthBook structure to hold market depth for delta updates
+type MarketDepthBook struct {
+	buys   map[string]*proto.PriceLevel
+	sells  map[string]*proto.PriceLevel
+	seqNum uint64
+}
+
 var (
-	ts         tcell.Screen
-	redStyle   tcell.Style
-	greenStyle tcell.Style
-	whiteStyle tcell.Style
-	market     *proto.Market
-	marketData *proto.MarketData
-
-	args struct {
-		gRPCAddress string
-		marketID    string
-	}
+	ts           tcell.Screen
```

Comment:
better make all of these as members of a struct, and the functions where they're used as methods of that struct type